app/screens/login_options/login_options.js

```diff
@@ -89,27 +89,31 @@ class LoginOptions extends PureComponent {
 
     renderGitlabOption = () => {
         const {config, serverVersion} = this.props;
-        const version = serverVersion.match(/^[0-9]*.[0-9]*.[0-9]*(-[a-zA-Z0-9.-]*)?/g)[0];
-        if (config.EnableSignUpWithGitLab === 'true' && semver.valid(version) && semver.gte(version, 'v3.10.0')) {
-            return (
-                <Button
-                    key='gitlab'
-                    onPress={() => preventDoubleTap(this.goToSSO, this, ViewTypes.GITLAB)}
-                    containerStyle={[GlobalStyles.signupButton, {backgroundColor: '#548'}]}
-                >
-                    <Image
-                        source={gitlab}
-                        style={{height: 18, marginRight: 5, width: 18}}
-                    />
-                    <Text
-                        style={[GlobalStyles.signupButtonText, {color: 'white'}]}
+        const match = serverVersion.match(/^[0-9]*.[0-9]*.[0-9]*(-[a-zA-Z0-9.-]*)?/g);
```

Comment:
I meant to mention this earlier, but we probably want to save this regex pattern somewhere instead of copying and pasting it everywhere